daemon/networkdriver/portallocator/portallocator.go

```diff
@@ -47,6 +48,8 @@ type ipMapping map[string]protoMap
 var (
 	ErrAllPortsAllocated = errors.New("all ports are allocated")
 	ErrUnknownProtocol   = errors.New("unknown protocol")
+	ErrPortRangeFallback = fmt.Errorf("using fallback port range: %d - %d", beginPortRange, endPortRange)
```

Comment:
At least this errors should be private. At max shouldn't be errors at all. There is no usage for them as errors.